src/main/webapp/static/css/styles.css

```diff
@@ -128,9 +128,7 @@ footer.page-footer .footer-copyright {
 .btn.tokenButton {
     border-radius: 32px;
     background: linear-gradient(135deg, #b39ddb 0%, #9575cd 100%); /* deep-purple lighten-3 --> #9575cd deep-purple lighten-2 */
-}
-.nav-wrapper .btn.tokenButton {
-    background: linear-gradient(135deg, rgba(255,255,255, .4) 0%, rgba(255,255,255, .2) 100%);
+    margin: 0 16px;
```

Comment:
```suggestion
```

This line can be removed because the CSS framework handles padding automatically for links within `<li>` elements.

<img width="204" alt="Screen Shot 2021-09-09 at 4 14 01 PM" src="https://user-images.githubusercontent.com/15718174/132684016-daa1ac23-eb9b-4c69-ba02-83555f2166d8.png">

<img width="352" alt="Screen Shot 2021-09-09 at 4 14 42 PM" src="https://user-images.githubusercontent.com/15718174/132684024-1f62a785-fc5b-4345-8110-6a6c5299379a.png">

Reply:
Done!